riff-raff.yaml

```diff
@@ -8,6 +8,7 @@ templates:
     type: autoscaling
     parameters:
       bucket: aws-frontend-artifacts
+      warmupGrace: 60
```

Comment:
I guess it's hard to measure these things but it'd be interesting to know if we could also get the same benefits with 30 seconds. Maybe not something to action now but it's perhaps an optimisation for later

Reply:
Yes, I did think 30 would be a good value to try out next. I was perhaps a bit impatient by going straight to 60 for testing.